src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -236,7 +236,7 @@ private function parseProperty(ResourceMetadata $resourceMetadata, PropertyMetad
                     return $data;
                 }
 
-                $data['subType'] = $subProperty['subType'];
+                $data['subType'] = $subProperty['dataType'];
```

Comment:
Feels wrong, maybe `$subProperty['subType'] ?? $subProperty['dataType']` but we need to check the consequences.